packages/google_maps_flutter/google_maps_flutter/lib/src/polygon.dart

```diff
@@ -179,6 +195,18 @@ class Polygon {
     }
     return result;
   }
+
+  dynamic _holesToJson() {
```

Comment:
We should be able to just return `List<dynamic>`

Reply:
done